src/openrct2-ui/windows/Cheats.cpp

```diff
@@ -474,34 +474,34 @@ class CheatsWindow final : public Window
             {
                 auto ft = Formatter::Common();
                 ft.Add<money64>(1000.00_GBP);
-                SetCheckboxValue(WIDX_GUEST_IGNORE_RIDE_INTENSITY, gCheatsIgnoreRideIntensity);
-                SetCheckboxValue(WIDX_DISABLE_VANDALISM, gCheatsDisableVandalism);
-                SetCheckboxValue(WIDX_DISABLE_LITTERING, gCheatsDisableLittering);
+                SetCheckboxValue(WIDX_GUEST_IGNORE_RIDE_INTENSITY, GetGameState().Cheats.IgnoreRideIntensity);
```

Comment:
The gamestate is already fetched on line 457, so you can just do
```suggestion
                SetCheckboxValue(WIDX_GUEST_IGNORE_RIDE_INTENSITY, gameState.Cheats.IgnoreRideIntensity);
```
Same for the other occurrences.